tools/templates/files/fileTemplates/Template Presentation Classes.kt.child.2.kt

```diff
@@ -8,7 +8,7 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.tooling.preview.PreviewParameter
 import io.element.android.libraries.designsystem.preview.ElementPreview
-import io.element.android.libraries.designsystem.preview.PreviewsDayNight
+import io.element.android.libraries.composeutils.annotations.PreviewsDayNight
```

Comment:
This has to be reordered, sorry it cannot be done automatically on the template. This line should be moved one line above.